test/Reader/ReaderTest.php

```diff
@@ -158,7 +158,7 @@ public function testImportsUriAndThrowsExceptionIfNotAFeed()
             $this->markTestSkipped('testImportsUri() requires a network connection');
         }
 
-        Reader\Reader::import('http://twitter.com/alganet');
+        Reader\Reader::import('http://example.com');
```

Comment:
Why the change in URL? When I ran these tests last week, the original URLs worked correctly for the test expectations...